test/topology_experimental_raft/test_topology_recovery_basic.py

```diff
@@ -24,6 +24,8 @@ async def test_topology_recovery_basic(request, manager: ManagerClient):
     cql = manager.cql
     assert(cql)
 
+    finish_writes_and_verify = await start_writes_to_cdc_table(cql)
```

Comment:
We call `reconnect_driver` below after restarting servers. Won't that make all writes used by `cql` fail, from that point onwards?